control_plane/src/compute.rs

```diff
@@ -291,6 +291,9 @@ impl PostgresNode {
         conf.append("max_connections", "100");
         conf.append("wal_sender_timeout", "0");
```

Comment:
In the previous PR @knizhnik set it to `10s` and wrote:

> With disabled wal_sender_timeout, walsender is not sending keep-alive messages, in response to which pageserver is able to inform about it's progress.

I haven't checked the code, though, but it seems that we have to set it to non-zero value here as well